test/Extension.Tests.ps1

```diff
@@ -47,8 +49,9 @@ Describe 'SecretManagement.LastPass tests' {
 
     It 'Can store a PSCredential secret' {
         $secretText = 'This is my pscredential secret'
-        $secret = [PSCredential]::new('myUser', ($secretText | ConvertTo-SecureString -AsPlainText))
+        $secret = [PSCredential]::new('myUser', ($secretText | ConvertTo-SecureString -AsPlainText -Force))
```

Comment:
-Force is superfluous for the newer PS version but still mandatory for 5.1. so I added it to be able to test with 5.1 (